src/features/community/CommunityGardenEntry.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from "react";
```

Comment:
Shouldnt CommunityGardenEntry and CommunityGardenModal files be placed in src\features\farming\town\components, in the same place as GoblinVillageEntry and GoblinVillageModal?
Those components are displayed on farming screen, so to prevent confusion its better to place them there.

Maybe even create subfolders communityGarden and goblinVillage.

Reply:
Or other way around, goblin files should be moved out. @adamhannigan whats your opinion on that?

Reply:
refactored and moved files to `src/features/farming/town/components` thanks!